test/extended/setup.sh

```diff
@@ -98,16 +131,27 @@ function os::test::extended::setup {
     # Similar to above check, if the XFS volume dir mount point exists enable
     # local storage quota in node-config.yaml so these tests can pass:
     if [ -d "/mnt/openshift-xfs-vol-dir" ]; then
-	# The ec2 images have have 1Gi of space defined; want to give /registry a good chunk of that
-	# to store the images created when the extended tests run
+    # The ec2 images have have 1Gi of space defined; want to give /registry a good chunk of that
+    # to store the images created when the extended tests run
       sed -i 's/perFSGroup: null/perFSGroup: 896Mi/' $NODE_CONFIG_DIR/node-config.yaml
     fi
     echo "[INFO] Using VOLUME_DIR=${VOLUME_DIR}"
 
-    # This is a bit hacky, but set the pod gc threshold appropriately for the garbage_collector test.
-    os::util::sed 's/\(controllerArguments:\ \)null/\1\n    terminated-pod-gc-threshold: ["100"]/' \
-      ${MASTER_CONFIG_DIR}/master-config.yaml
+    # Update pod gc threshold appropriately for the garbage_collector test.
+    cp ${MASTER_CONFIG_DIR}/master-config.yaml ${MASTER_CONFIG_DIR}/master-config.orig.yaml
```

Comment:
This is unrelated to audit, right? Maybe should be separate commit.

Also, unless we *want* to keep both versions of this config, clean up the useless one.